apps/3002-checkout/next.config.js

```diff
@@ -1,12 +1,24 @@
+const { withNx } = require('@nrwl/next/plugins/with-nx');
```

Comment:
apps/3002-checkout should use old outdated `antd@4.20.0`
It helps to check that module resolution with different `antd` versions works correctly

Reply:
Ok I'll try to resolve it tomorrow.
I also want to use different `nextjs` versions between apps to test their compatibility.

Because in real scenarios (like mine where I have 20 apps) it's very difficult to update all applications simultaneously.

Reply:
Okay, so I have figured it a way to resolve the issue.. I'm working on it.

Reply:
Fixed 👍